pycbc/__init__.py

```diff
@@ -96,13 +95,9 @@ def sig_handler(signum, frame):
 
     signal.signal(signal.SIGUSR1, sig_handler)
 
-    if not verbose:
```

Comment:
I think this doesn't take into account if `--verbose` uses `action='store_true'` and ends up as `False`, which worked before.

I suggest to keep the `if not verbose: elif`, or find every instance which uses `init_logging` and verbose as `store_true`.

The second is probably better but time consuming, the first would be easier to implement.

I would like to standardise logging and versioning throughout the code, which would involve the first option as it has become quite inconsistent, but that is a medium-term objective

Reply:
Wait - ignore that, `int(False)` is zero

Reply:
ie one of the laundry list items in https://github.com/gwastro/pycbc/issues/4483 .. 
Resolvable, I guess